module/applications/actor/character-sheet-2.mjs

```diff
@@ -84,7 +84,8 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
     const hp = foundry.utils.getProperty(data ?? {}, "system.attributes.hp.value");
     if ( isUpdate && (hp === 0) ) this._toggleDeathTray(true);
     const [bastion] = this.element.find('nav.tabs [data-tab="bastion"]');
-    if ( bastion ) bastion.toggleAttribute("hidden", this.object.system.details.level < 5);
+    const { enabled } = game.settings.get("dnd5e", "bastionConfiguration");
+    if ( bastion ) bastion.toggleAttribute("hidden", (this.actor.system.details.level < 5) || !enabled);
```

Comment:
Would be good to have this (the level requirement) in the config rather than hardcoded. It doesn't need to be a setting, just a simple numeric property. To make it simple, any module or script changing the level requirement should also be required to change the localization to match.

Reply:
We have `CONFIG.DND5E.facilities.advancement` which is closely tied to this number. Would using the min of that variable work?

Reply:
Didn't realize we had that. Seems good.

Reply:
OK I will make that change then